modules/xmpp/JingleSessionPC.js

```diff
@@ -290,18 +324,32 @@ export default class JingleSessionPC extends JingleSession {
                 this,
                 this.peerconnection.iceConnectionState);
             switch (this.peerconnection.iceConnectionState) {
+            case 'checking':
+                this._iceCheckingStartedTimestamp = now;
```

Comment:
Do we know how they deal with reconnects (i.e. if they trigger checking state again)?